mmengine/optim/optimizer/optimizer_wrapper.py

```diff
@@ -161,7 +161,7 @@ def __init__(self,
         # the loss factor will always be the same as `_accumulative_counts`.
         self._remainder_counts = -1
 
-    def update_params(self, loss: torch.Tensor) -> None:
+    def update_params(self, loss: torch.Tensor, **kwargs) -> None:
```

Comment:
```suggestion
    def update_params(self, loss: torch.Tensor, step_kwargs=None, zero_grad=None) -> None:
```

Consider `update_params` will call `step` and `zero_grad` in order, it could be better to provide `step_kwargs` and `zero_kwargs` arguments. 

BTW, we should also update the description of the argument in docstring,